lua/starfall/libs_sh/builtins.lua

```diff
@@ -637,6 +641,10 @@ else
 	-- @param number mtype How the message should be displayed. See http://wiki.facepunch.com/gmod/Enums/HUD
 	-- @param string text The message text.
 	function builtins_library.printMessage(mtype, text)
+		if instance.player == SF.Superuser then
+			LocalPlayer():PrintMessage(mtype, text)
```

Comment:
Need to move `checkluatype(text, TYPE_STRING)` to the top of this function

Reply:
Can this if case also be an elseif, after checking if instance.player is localplayer